Dockerfile.ppc64le

```diff
@@ -0,0 +1,100 @@
+# This file describes the standard way to build Docker, using docker
+#
+# Usage:
+#
+# # Assemble the full dev environment. This is slow the first time.
+# docker build -t docker -f Dockerfile.gccgo .
+#
+
+FROM ppc64le/gcc:5.2
+ENV CGO_ENABLED 1
+ENV USE_GCCGO  1
+
+# Packaged dependencies
+RUN apt-get update && apt-get install -y \
+	apparmor \
+	aufs-tools \
+	btrfs-tools \
+	build-essential \
+	curl \
+	git \
+	iptables \
+	net-tools \
+	libapparmor-dev \
+	libcap-dev \
+	libsqlite3-dev \
+	mercurial \
+	parallel \
+	python-mock \
+	python-pip \
+	python-websocket \
+	--no-install-recommends
+
+RUN rm -rf /usr/local/lvm2
+RUN git clone --no-checkout git://git.fedorahosted.org/git/lvm2.git /usr/local/lvm2 && cd /usr/local/lvm2 && git checkout -q v2_02_103
+RUN curl -o /usr/local/lvm2/autoconf/config.guess 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD'
+RUN curl -o /usr/local/lvm2/autoconf/config.sub 'http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD'
+
+# Compile and install lvm2
+RUN cd /usr/local/lvm2 \
+	&& ./configure --enable-static_link \
+	&& make device-mapper \
+	&& make install_device-mapper
+# see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
+
+# Install lxc
+ENV LXC_VERSION 1.1.2
+RUN mkdir -p /usr/src/lxc \
+	&& curl -sSL https://linuxcontainers.org/downloads/lxc/lxc-${LXC_VERSION}.tar.gz | tar -v -C /usr/src/lxc/ -xz --strip-components=1
+RUN cd /usr/src/lxc \
+	&& ./configure \
+	&& make \
+	&& make install \
+	&& ldconfig
```

Comment:
How would you like to solve this issue?

> On Nov 13, 2015, at 3:35 PM, Tianon Gravi <notifications@github.com> wrote:
> 
> In Dockerfile.ppc64le <https://github.com/docker/docker/pull/17459#discussion_r44830804>:
> 
> > +# Compile and install lvm2
> > +RUN cd /usr/local/lvm2 \
> > +	&& ./configure --enable-static_link \
> > +	&& make device-mapper \
> > +	&& make install_device-mapper
> > +# see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
> > +
> > +# Install lxc
> > +ENV LXC_VERSION 1.1.2
> > +RUN mkdir -p /usr/src/lxc \
> > +	&& curl -sSL https://linuxcontainers.org/downloads/lxc/lxc-${LXC_VERSION}.tar.gz | tar -v -C /usr/src/lxc/ -xz --strip-components=1
> > +RUN cd /usr/src/lxc \
> > +	&& ./configure \
> > +	&& make \
> > +	&& make install \
> > +	&& ldconfig
> This is a perfect example of why I'm really, really, really uncomfortable with having additional copies of the main Dockerfile. This is already outdated, and even if it'd merged before the previous one that removed this, it likely wouldn't have been updated, because most contributors don't even think to check other files when need to update the main Dockerfile (or any other file like this).
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/docker/docker/pull/17459/files#r44830804>.
>